lib/scanner/adapters/block-device.ts

```diff
@@ -59,7 +59,12 @@ export class BlockDeviceAdapter extends Adapter {
 	private running = false;
 	private ready = false;
 
-	constructor(public includeSystemDrives: () => boolean = () => false) {
+	constructor(
+		public includeSystemDrives: () => boolean = () => false,
+		private unmountOnSuccess = false,
+		private oWrite = false,
+		private oDirect = true,
```

Comment:
Can you make these an options object please, because something like `new scanner.adapters.BlockDeviceAdapter(() => false, false, true, true),` is very hard to understand